### PR TITLE
adding video to Dec 6th

### PR DESCRIPTION
**Pull request by OliveCannon**
adding video to Dec 6th

### CODE REVIEW

**Review by clcoffey — APPROVED** [review]
thanks Olive

File: content/en/events/meetup-2022-12-14.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ redirect: "https://bit.ly/ISCEoY22"
 image: "/images/events/meetup-2022-12-14.png"
 date: 2022-12-14
 publishDate: 2022-11-27
-
+youtubeLink: "https://youtu.be/r8Ce7GlwBeA"
```
Comment (spier) [inline content/en/events/meetup-2022-12-14.md]:
```suggestion
```

**Review by spier — APPROVED** [review]
@OliveCannon you said you were changing the Dec 6th event.
However you also added a video to the Dec 14th event (which hasn't happened yet).

I figured that that was a mistake and have removed that video for the event on the 14th.

**Comment by spier** [comment]
@OliveCannon can you confirm that this now looks as desired here?
https://innersourcecommons.org/events/

**Pull request merged**